src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+class TodoForm extends Component{
+    constructor(props) {
+        super(props);
+        this.state = {value: ''};
+    
+        this.handleChange = this.handleChange.bind(this);
+      }
+    
+      handleChange(event) {
+        this.setState({value: event.target.value});
+      }
+    render() {
+       
+        return (
+          <form onSubmit={(event) => {this.props.cb_submit(this.state.value); event.preventDefault();this.setState({value:''});}}>
+            <label>
+              Name:
+              <input type="text" value={this.state.value} onChange={this.handleChange} />
```

Comment:
Instead of using an inline function definition for your `onSubmit` event handler, you can define this function separately and then send it in as a callback rather than condensing what should be multiple lines of code inside a single tag..